Assets/Python/GreatPeople.py

```diff
@@ -2466,6 +2501,7 @@ def setup():
 		"John Stott", # 20th
 	],
 	iGreatArtist : [
+		u"Ælfric of Eynsham", # 10th
```

Comment:
The game cannot render this character, so it has to be Aelfric

Reply:
"æ" is present in many Scandinavian names already; are you sure it can't be rendered?

Reply:
Oh, I checked again and it is.

Reply:
I removed the Æ before writing my comment, so you might want to put it back in.